src/functions/read.ts

```diff
@@ -27,16 +28,23 @@ interface createReadOptions {
    * @default `minimal`
    */
   createMode?: RCGenerationMode | RCGenerationMode[];
+
+  /**
+   * RC automatic caching option. This option allows to cache a configuration passed in parameter.
+   *
+   * @default false
+   */
+  isMemoize?: boolean ;
 }
 
-export type readOptions = RequireAtLeastOne<createReadOptions, "createIfDoesNotExist" | "createMode">;
+export type readOptions = RequireAtLeastOne<createReadOptions, "createIfDoesNotExist" | "createMode" | "isMemoize">;
```

Comment:
No that's not ok, that's not a required argument. No need to update readOptions type here.